### DISCUSSION TITLE
Suggest to start on random available port if the default is in use.

### DISCUSSION THREAD

**Discussion by pinikeizman** [vercel/next.js#27728] [Ideas]
### Describe the feature you'd like to request

When I create a new app using `create-next-app` and run it with `yarn start` and another process is listening on the default port I get this error:
```
 marketplace git:(main) yarn start
yarn run v1.22.10
$ next start
Error: listen EADDRINUSE: address already in use 0.0.0.0:3000
    at Server.setupListenHandle [as _listen2] (net.js:1318:16)
    at listenInCluster (net.js:1366:12)
    at doListen (net.js:1503:7)
    at processTicksAndRejections (internal/process/task_queues.js:81:21) {
  code: 'EADDRINUSE',
  errno: -48,
  syscall: 'listen',
  address: '0.0.0.0',
  port: 3000
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command
```
I think this is a bed developer experience.

### Describe the solution you'd like

Suggest the user to use a different but available port.

### Describe alternatives you've considered

I can't think on a better alternative.

**Comment by vogelino** [comment]
I use tmux to manage/code on different nextjs websites simultaneously and I cannot run two websites simultaneously because of this issue. I would love nextjs to just open the dev mode on the next available port.
